straxen/analyses/waveform_plot.py

```diff
@@ -338,3 +344,271 @@ def plot_wf(st: strax.Context,
                          vmin=1 if plot_log else None,
                          log_scale=plot_log,
                          label='Area per channel [PE]')
+
+
+# Default attributes to display in the event_display (looks little
+# complicated but just repeats same fields for S1 S1)
+# Should be of form as below where {v} wil be filled with the value of
+# event['key']:
+#  (('key', '{v} UNIT'), ..)
+PEAK_DISPLAY_DEFAULT_INFO = sum([[(k.format(i=s_i), u) for k, u in
+                                  (('cs{i}', '{v:.2f} PE'),
+                                   ('s{i}_area', '{v:.2f} PE'),
+                                   ('alt_cs{i}', '{v:.2f} PE'),
+                                   ('s{i}_n_channels', '{v}'),
+                                   ('s{i}_area_fraction_top', '{v:.2f}'),
+                                   ('s{i}_range_50p_area', '{v:.1f}'),
+                                   )] for s_i in (1, 2)], [])
+EVENT_DISPLAY_DEFAULT_INFO = (('time', '{v} ns'),
+                              ('endtime', '{v} ns'),
+                              ('event_number', '{v}'),
+                              ('x', '{v:.2f} cm'),
+                              ('y', '{v:.2f} cm'),
+                              ('z', '{v:.2f} cm'),
+                              ('r', '{v:.2f} cm'),
+                              ('theta', '{v:.2f} rad'),
+                              ('drift_time', '{v} ns'),
+                              ('alt_s1_interaction_drift_time', '{v} ns'),
+                              ('alt_s2_interaction_drift_time', '{v} ns')
+                              )
+
```

Comment:
Would not be a dictionary simpler?

Reply:
Okay, I checked it out, the format would be very similar but we would have a mutable default which then requires if `display_peak_info is None:` et cetera. Therefore, I keep it as it is.
```python
PEAK_DISPLAY_DEFAULT_INFO = {}
_ = [PEAK_DISPLAY_DEFAULT_INFO.update(
    {k.format(i=i): v for k, v in
     {'cs{i}': '{v:.2f} PE',
      's{i}_area': '{v:.2f} PE',
      'alt_cs{i}': '{v:.2f} PE',
      's{i}_n_channels': '{v}',
      's{i}_area_fraction_top': '{v:.2f}',
      's{i}_range_50p_area': '{v:.1f}'}.items()}) for i in (1, 2)]
PEAK_DISPLAY_DEFAULT_INFO
EVENT_DISPLAY_DEFAULT_INFO = {
    'time': '{v} ns',
    'endtime': '{v} ns',
    'event_number': '{v}',
 'x': '{v:.2f} cm',
 'y': '{v:.2f} cm',
 'z': '{v:.2f} cm',
 'r': '{v:.2f} cm',
 'theta': '{v:.2f} rad',
 'drift_time': '{v} ns',
 'alt_s1_interaction_drift_time': '{v} ns',
 'alt_s2_interaction_drift_time': '{v} ns'}
 ```